recipes/mp-units/2.0.0/conanfile.py

```diff
@@ -81,11 +73,13 @@ def loose_lt_semver(v1, v2):
             raise ConanInvalidConfiguration(
                 f"{self.ref} requires at least {compiler} {min_version} ({compiler.version} in use)"
             )
-        
-        # Note that apple-clang and msvc are disabled for now, their C++ 20 implementations are not up to speed
+
+        # Note that apple-clang is disabled for now, its C++ 20 implementation are not up to speed
```

Comment:
This comment looks inconsistent with this check. Perhaps this comment can be removed since the message in the error has a similar description.